Formula/terragrunt.rb

```diff
@@ -15,7 +15,7 @@ class Terragrunt < Formula
   end
 
   depends_on "go" => :build
-  depends_on "terraform"
+  depends_on "tfenv" => :test
```

Comment:
Based on my suggested test below, we would want 
```suggestion
  depends_on "terraform" => :test
```